code/modules/admin/verbs/modifyvariables.dm

```diff
@@ -142,15 +142,33 @@ var/list/VVckey_edit = list("key", "ckey")
 		if(confirm != "Continue")
 			return
 
-	var/list/names = sortList(L)
+	var/assoc = 0
+	if(L.len > 0)
+		var/a = L[1]
+		if(!isnum(a) && L[a] != null)

```

Comment:
Just finished porting this over and I had a few problems with this line. "!isnum(a)" allows types and other non-numerical entries to go through. This causes "L[a]" to runtime due to a bad index. Changing "!isnum(a)" to "istext(a)" seemed to fix the issue.